doc/conf_ref.rst

```diff
@@ -721,6 +736,8 @@ configuration.
     for any reason. This option doesn't affect skipping of unavailable packages after dependency
     resolution. To check inaccessibility of repository use it in combination with
     :ref:`refresh command line option <refresh_command-label>`. The default is ``False``.
+    Note this option in particular :ref:`can be set in your configuration file
```

Comment:
`Note this option in particular` sounds strange to me.
What about:
```
Note this option :ref:`can be set in your configuration file in particular distributions <conf_distribution_specific-label>`.
```

Reply:
@lukash What do you think?

Reply:
Well.. I think my use of _in particular_ is ok, it is meant to highlight that this particular (:slightly_smiling_face:) option is known to be overridden in distros.

I think saying _in particular distributions_ is a strange formulation (although probably not strictly wrong?), I think in this case saying _in some distributions_ would be better, but I just used _in your distribution_, because I'm kind of assuming that's the context for the reader. (and now I'm thinking if it always is)